data/mods/Xedra_Evolved/spells/inventor_spells.json

```
@@ -52,6 +52,7 @@
     "effect": "attack",
     "shape": "cone",
     "valid_targets": [ "ally", "hostile", "ground" ],
+    "skill": "deduction",
```

```suggestion
    "skill": "deduction"
```
I think the tests are frozen on this so I'm hard reseting them by adding new commits.